tests/main/interfaces-ros-snapd-support/api-apps-client/bin/api-apps-client.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
```

Comment:
I wonder if we have a way to use `snap debug api` from snaps in a way that would allow us to test interfaces. Technically everything you do here is also done by `snap debug api` in either GET or POST mode.

Reply:
I didn't know about `snap debug api`, but how would I use that in a snap? should I `stage-snap` `snapd`?

Reply:
I don't think it's worth it to add snap just to the debug API command. There's too much mechanical changes required if you wanted to do it this way (snapcraft + store snap + request permission for installation from the store-reviewers). You could theoretically try to copy the local /usr/bin/snap binary into $SNAP_COMMON before running the snap, but I don't mind having this trivial python client tbh.